extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTableWriter.java

```diff
@@ -90,13 +90,37 @@ enum CacheTags {
      * @param <B>
      */
     interface TransferObject<B> extends SafeCloseable {
```

Comment:
Once we complete the review and before merging, I am thinking of reorganizing this file by moving `TransferObject` closer to its child classes and reordering all methods in child classes to be in same order as this interface.

Reply:
Sounds great. Do it as a single, isolated commit, please.

Reply:
Try not to do anything that will introduce even more conflicts with https://github.com/deephaven/deephaven-core/pull/4479